deepchem/models/torch_models/dmpnn.py

```diff
@@ -10,7 +10,10 @@
 
 from deepchem.feat import GraphData
 from deepchem.data import Dataset
-from typing import Union, List, Sequence, Optional, Iterable, Tuple
+from typing import Any, Union, List, Sequence, Optional, Iterable, Tuple
+
+pytorch_geometric_data = Any  # type annotation for torch_geometric.data.Data
```

Comment:
Why can;t we directly use torch_geometric.data.Data?

Reply:
To solve the import checking error, where pytorch-geometric cannot be imported, I put the import statement under Try-except. Hence when importing pytorch-geometric is not possible, the type notation **torch_geometric.data.Data** would be undefined. So, I made new type annotations for data and batch. 

This is the error that was solved by doing the above: https://github.com/deepchem/deepchem/runs/8231081333?check_suite_focus=true